hera_pspec/pspecdata.py

```diff
@@ -1805,9 +1838,11 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None, input_data_weight='identit
             where the first index is for the Left-Hand dataset and second index
             is used for the Right-Hand dataset (see above).
 
-        pols : length-2 tuple of strings or integers or list of length-2 tuples of strings or integers
+        pols : length-2 tuple of strings or integers or list of length-2 
+            tuples of strings or integers
             Contains polarization pairs to use in forming power spectra
-            e.g. ('XX','XX') or [('XX','XX'),('XY','YX')] or list of polarization pairs.
+            e.g. ('XX','XX') or [('XX','XX'),('pI','pI')] or list of 
+            polarization pairs.
             Only auto/equal polarization pairs are implemented at the moment.
```

Comment:
although you didn't add this, maybe good to get this consistent: per UW requests perhaps we change this to `Only identical polarization pairs...`?

Reply:
I rewrote this as it's a bit confusing anyway.